src/maxdiffusion/configs/base_xl.yml

```diff
@@ -79,7 +79,7 @@ ici_tensor_parallelism: 1
 
 # Dataset
 # Replace with dataset path or train_data_dir. One has to be set.
-dataset_name: ''
+dataset_name: 'diffusers/pokemon-gpt4-captions' 
```

Comment:
Do we want to separate generate and train config files?

Reply:
Its getting difficult to manage all the different config files so I'm trying to consolidate them all into single files similar to how maxtext is doing it.